js-core/homeworks/contact-list/index.html

```diff
@@ -1,11 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link href='src/main.css' rel='stylesheet'>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
     <meta charset="UTF-8">
-    <title>Home work 1</title>
+    <title>Phone App</title>
 </head>
 <body>
-
+<div class='main'>
+    <div class="app-header">
+        My Phone App
+    </div>
+    <div class='app-content'>
```

Comment:
you have to use double quotes for attributes, it could cause problems in some browser